src/navigation/routerConfig.ts

```diff
@@ -1,2 +1,32 @@
-// router cfg will go here
-export const ROUTER_CONFIG = {}
+import DirectionView from "../modules/direction/components/DirectionView";
+import MainView from "../modules/main/components/MainView";
+import { RenderRoutes } from "./Router";
+
```

Comment:
interface for the route config will be nice:
```suggestion

export interface IRouterConfig {
  component: React.ComponentType, // double check
  path: string,
  key: string,
  exact?: boolean,
  routes?: IRouterConfig[],
}
```